components/CustomMarker.js

```diff
@@ -66,7 +64,7 @@ const CustomMarker = ({ data }) => {
             visible={modalVisible}
           >
             <View style={styles.modalContainer}>
-              <FetchS3Image imageURL={parsedData.image} />
+              <FetchS3Image imageURL={parsedData.observationImageURL} />
```

Comment:
we got images now for real they didn't believe in us, they don't want us to have images in our customPins. we da best music . you ever play rugby? Sunday brunch **😬😬😬😬** Sunday breakfast i call it
![Poster_Book_Kant_Universal History](https://github.com/UCBoulder/HabitatUI/assets/42285595/8e246164-f86b-422f-b667-581c7940d59b)